src/Microsoft.FeatureManagement/FeatureManagementOptions.cs

```diff
@@ -22,5 +26,7 @@ public class FeatureManagementOptions
         /// The default value is true.
         /// </summary>
         public bool IgnoreMissingFeatures { get; set; } = true;
+
+        internal ICollection<Func<IServiceProvider, ITelemetryPublisher>> telemetryPublisherFactories;
```

Comment:
I think it would be valuable to have a summary here as well for clarity in the future.

Reply:
Should be a property, and start with an upper case.